docs/tutorials/getting-started-with-flask.rst

```diff
@@ -0,0 +1,114 @@
+Getting started with Rockcraft for Flask
```

Comment:
Would it be more appropriate to say, "Getting started with Rockcraft for Flask applications"? Or has "applications" been explicitly removed to reduce the title size?

Reply:
I'm happy to include it, but yeh it gets a bit long

Reply:
I don't have a strong opinion either way. I think the title makes sense with or without "applications"; I mostly wanted to make sure that was the case